scipy/stats/_mannwhitneyu.py

```diff
@@ -223,21 +214,15 @@ def _mwu_input_validation(x, y, use_continuity, alternative, axis, method):
     return x, y, use_continuity, alternative, axis_int, method
 
 
-def _tie_check(xy):
-    """Find any ties in data"""
-    _, t = np.unique(xy, return_counts=True, axis=-1)
-    return np.any(t != 1)
-
-
-def _mwu_choose_method(n1, n2, xy, method):
+def _mwu_choose_method(n1, n2, ties, method):
```

Comment:
This function does not use the argument `method`. So, this can be removed?

Reply:
I suppose so. I guess it was like that before.